Dockerfile

```diff
@@ -442,12 +442,10 @@ ENTRYPOINT ["/bin/bash"]
 FROM components AS run
 
 # nuget-inspector
-ARG NUGET_INSPECTOR_VERSION=0.7.0
-RUN mkdir -p nuget-inspector \
-    && curl -L https://github.com/nexB/nuget-inspector/releases/download/v${NUGET_INSPECTOR_VERSION}/nuget-inspector-v${NUGET_INSPECTOR_VERSION}-linux-x64.tar.gz \
-    | tar -C ./nuget-inspector -xz
+ARG NUGET_INSPECTOR_VERSION=0.7.1
+RUN curl -L https://github.com/nexB/nuget-inspector/releases/download/v${NUGET_INSPECTOR_VERSION}/nuget-inspector-v${NUGET_INSPECTOR_VERSION}-linux-x64.tar.gz \
+    | tar -xz
 COPY --chown=$USERNAME:$USERNAME nuget-inspector /opt/nuget-inspector
-# RUN chmod 755 /opt/nuget-inspector/nuget-inspector
```

Comment:
This change should be amended to the first commit.

Reply:
done